doc/specs/#1620 - Installer selector on multiple matches.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Darshan Rander @SirusCodes
+created on: 2022-10-29
+last updated: 2022-10-29
+issue id: 1620
+---
+
+# Spec Title
+
+For [#1620](https://github.com/microsoft/winget-cli/issues/1620)"
+
+## Abstract
+
+The winget.exe client simply lists all the matches and if the user wants to install the app they need to re-run the command with Id.
+
+The UX can be improved here by giving users an option to select the package instead of asking them to re-run the command with Id.
+
+## Inspiration
+
+It is not a good experience to copy Id and then run the command again.
+
+## Solution Design
+
+If we find multiple packages then the user should get a selector.
+
+winget should ask for user input, for example -
+
+```pwsh
+> winget install Powershell
+Multiple packages found matching.
+No. Name       Id                   Source
+-------------------------------------------
+ 1  PowerShell 9MZ1SNWT0N5D         msstore
+ 2  PowerShell Microsoft.PowerShell winget
+
+Please select the packages (eg. 2): <user-input>
+
+Installing <user-input>...
+```
+
+> `<user-input>` - is the input by user
+
+After the user will give the input the installation should start.
+
+## UI/UX Design
+
+The idea is explained in [Solution Design](#solution-design).
+
+## Capabilities
+
+### Accessibility
+
+It should not have any direct impact on screen readers.
+
+### Security
+
+It should not have any security issues.
+
+### Reliability
+
+This will help the users to have a better experience and would decrease the chance of human error while copy-pasting the Id.
```

Comment:
Will there need to be any measures taken against mistyping the selection? It is a lot easier to accidentally type `2` instead of `3`  than it is to enter an incorrect package ID.

Reply:
Yeah, it's something which is something that can happen. Adding another prompt to take the confirmation saying "Do you want to install XYZ package? (Y/n): ". But again, it's an extra step.

I would add this in the in doc. Don't have any idea how we can handle it

Reply:
Perhaps it's a future consideration